Cargo.toml

```diff
@@ -33,9 +33,10 @@ members = [
   "crates/swift-bridge-cli",
   "crates/swift-bridge-ir",
   "crates/swift-bridge-macro",
-  "crates/swift-integration-tests",
+
+  "crates/swift-integration-tests","crates/swift-bridge-build/tests/sample-project",
```

Comment:
Instead of `sample-project` we should name the crate `test-swift-packages`

Reply:
Oh sorry I meant to delete this message! `sample-project` was a fine name.

Whatever you want to go with is fine here.

Reply:
Oh no worry ignore this. This actually is the test. I think?

Reply:
This is the Cargo project that is used in the test.